src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/dbscripts/InternalScriptLoader.java

```diff
@@ -21,38 +21,50 @@
 import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 import com.google.common.io.CharStreams;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URL;
+import java.util.List;
+import java.util.Set;
 import java.util.function.Supplier;
 
 public class InternalScriptLoader implements ScriptLoader {
 
+  private static final Set<String> scriptNames =
+      ImmutableSet.of(
+          "all_ri_children",
+          "all_ri_parents",
+          "columns",
+          "diskspace",
+          "functioninfo",
+          "indices",
+          "partitioning_constraints",
+          "query_references",
+          "querylogs",
+          "roles",
+          "sql_logs",
+          "stats",
+          "tableinfo",
+          "tablesize",
+          "temp_tables",
+          "users");
+
   @Override
   public ImmutableMap<String, Supplier<String>> loadScripts() {
-    return ImmutableList.of(
-            "all_ri_children",
-            "all_ri_parents",
-            "columns",
-            "diskspace",
-            "functioninfo",
-            "indices",
-            "partitioning_constraints",
-            "query_references",
-            "querylogs",
-            "roles",
-            "sql_logs",
-            "stats",
-            "tableinfo",
-            "tablesize",
-            "temp_tables",
-            "users")
-        .stream()
+    return scriptNames.stream()
         .collect(
             ImmutableMap.toImmutableMap(Functions.identity(), key -> scriptLoader(key + ".sql")));
   }
 
+  @Override
+  public ImmutableMap<String, ImmutableList<String>> getSortingColumnsMap() {
+    return new ImmutableMap.Builder<String, ImmutableList<String>>()
+        .put("querylogs.sql", ImmutableList.of("\"StartTime\""))
```

Comment:
It depends on how will we approach sql logs and query texts merge, but just a note for the next PRs – we may want to sort sql_logs too and order of querylogs and sql_logs should be consistent to merge data effectively:
* StartTime/CollectTimeStamp ASC
* QueryID ASC
* (sql_logs specific) SqlRowNo DESC

Reply:
Thanks for the heads-up - I'll keep that in mind.